app/src/main/java/org/oppia/android/app/options/TextSizeRadioButtonListener.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.android.app.options
+
+/**
+ * Listener for when the reading text size is selected from the [ReadingTextSizeFragment]
```

Comment:
Update to `/** Listener for when the reading text size is selected from the [ReadingTextSizeFragment]. */`